app/src/worker/alarmClock.js

```diff
@@ -1,34 +1,33 @@
-import StorageService from "../service/storageService";
+import StorageService from '../service/storageService'
 
 // eslint-disable-next-line no-unused-vars
-let clock;
-let currentTime;
-let alarms;
+let clock
+let currentTime
+let alarms
 
-export function init(){
-    clock = setInterval(() => setCurrentTime(), 1000);
+export function init () {
+  clock = setInterval(() => setCurrentTime(), 1000)
 }
 
-async function loadAlarmsFromDb() {
-  const config = await StorageService.loadConfig();
-  alarms = config.alarms;
+async function loadAlarmsFromDb () {
+  const config = await StorageService.loadConfig()
+  alarms = config.alarms
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/ruizivo/ahgora-login/pullRequest?prid=9873926)
```suggestion
  alarms = config.alarms;
```